### PR TITLE
Docs: Creating-text: Suggest three-mesh-ui first.

### PR DESCRIPTION
**Pull request by gonnavis**
Related issue: -

**Description**

I tried to find that [three-mesh-ui](https://github.com/felixmariotto/three-mesh-ui) is much more convenient to use than [three-bmfont-text](https://github.com/Jam3/three-bmfont-text), and it provides `ordinary .js files` and `import` and `require` simultaneously.
In contrast, three-bmfont-text only provides `require` which I think is only used for nodejs and depends on `global.THREE`, in fact, I even failed to use three-bmfont-text successfully.
So I think it would be better to recommend three-mesh-ui first to beginners.

### CODE REVIEW

**Comment by donmccurdy** [comment]
This is updating a section on bitmap fonts — is three-mesh-ui using that method? Sections (4) and (5) are separated because there are situations where either method might be needed. A library more similar to three-bmfont-text, and probably also easier to use, is [three-troika-text](https://github.com/protectwise/troika/blob/master/packages/troika-three-text/README.md).

**Comment by gonnavis** [comment]
Hello @donmccurdy, [three-mesh-ui](https://github.com/felixmariotto/three-mesh-ui) can use the font file generated from [your site](https://msdf-bmfont.donmccurdy.com/), and it's [doc also say this](https://github.com/felixmariotto/three-mesh-ui/wiki/Creating-your-own-fonts).

I think it using bitmap fonts, isn't it?

[demo](https://raw.githack.com/gonnavis/three.js/three-mesh-ui/examples/three-mesh-ui.html)

But I found out that it doesn't support spaces unexpectedly. I didn't find this problem yesterday because all chinese characters had no spaces. I need to raise an issue first.

**Comment by gonnavis** [comment]
@donmccurdy  Confirmed with three-mesh-ui author that it using bitmap fonts https://discourse.threejs.org/t/how-to-use-bmfont/21206/10 .

I'll try three-troika-text too, but you have confirmed. Could list them all like `three-troika-text, three-troika-text or three-bmfont-text` .

**Comment by felixmariotto** [comment]
@gonnavis I've just fixed your spacing problem, check your issue on Github.   
Thank you for proposing to add three-mesh-ui as a suggestion to add text into three.js.

@donmccurdy what makes you think that troika is easier to use than [three-mesh-ui](https://github.com/felixmariotto/three-mesh-ui) ?

**Comment by gonnavis** [comment]
@felixmariotto Thanks for fix! I confirmed that this ["Hello world" demo](https://raw.githack.com/gonnavis/three.js/three-mesh-ui/examples/three-mesh-ui_hellospaceworld.html) works now.

**Comment by donmccurdy** [comment]
Ok, as long as it's also a BMFont-based solution I'm fine with listing them both. I don't feel strongly that three-troika-text needs to be added, as I haven't used it myself, but we can always consider that later.

**Review by donmccurdy — APPROVED** [review]

**Review by Mugen87 — APPROVED** [review]

File: docs/manual/ar/introduction/Creating-text.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h2>5. خطوط نقطية (Bitmap Fonts)</h2>
 		<div>
 			<p>
 				تسمح BMFonts (الخطوط النقطية) بدمج الصور الرمزية في BufferGeometry واحد. يدعم عرض BMFont التفاف الكلمات ، وتباعد الأحرف ، وتقنين الأحرف ، وحقول المسافة الموقعة مع المشتقات القياسية ، وحقول المسافة الموقعة متعددة القنوات ، والخطوط متعددة الأنسجة ، والمزيد.
-				انظر [link:https://github.com/Jam3/three-bmfont-text three-bmfont-text].
+				انظر [link:https://github.com/Jam3/three-bmfont-text three-bmfont-text] or [link:https://github.com/felixmariotto/three-mesh-ui three-mesh-ui].
```
Comment (mrdoob) [inline docs/manual/ar/introduction/Creating-text.html:92]: I think you need to change the order on this one?

Comment (gonnavis) [reply docs/manual/ar/introduction/Creating-text.html]: I know that Arabic is read from right to left, but I'm very confused by the different Editing and Viewing order with it.
If look at the overall right-to-left order, it is right now, so I deliberately used this order which different from `en` and `zh`.
But from the single piece of English `three-bmfont-text or three-mesh-ui`, the order is wrong.
image
image

I don't know which one is correct, maybe all wrong.
So I'll changed to using Arabic word of `or`.
image
image

image

Comment (mrdoob) [reply docs/manual/ar/introduction/Creating-text.html]: That's a rabbit hole I prefer not to venture into 😅

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**